templates/install.rdf.mustache

```diff
@@ -7,10 +7,10 @@
     <em:multiprocessCompatible>{{addon.multiprocessCompatible}}</em:multiprocessCompatible>
     <em:hasEmbeddedWebExtension>{{addon.hasEmbeddedWebExtension}}</em:hasEmbeddedWebExtension>
     <em:unpack>false</em:unpack>
-    <em:version>{{addon.version}}</em:version>
+    <em:version>{{version}}</em:version>
     <em:name>{{addon.name}}</em:name>
-    <em:description>{{addon.description}}</em:description>
-    <em:creator>{{addon.creator}}</em:creator>
+    <em:description>{{description}}</em:description>
+    <em:creator>{{author}}</em:creator>
```

Comment:
These 3 values `<em:version>`, `<em:description>`, and `<em:creator>` should now get their values from the main package.json values.